README.md

```diff
@@ -1,216 +1,265 @@
+## Examine the project structure
+
+Open the project in Android Studio and switch the view from **Android** to **Project** to see all the files and targets belonging to the project:
 
 <img src="readme_images/open_project_view.png" height="300px">
 
 Your Compose Multiplatform project includes three modules:
 
-### `shared`
-This Kotlin module that contains the logic common for both Android and iOS applications – the code you share between platforms.
+### shared
+
+This is a Kotlin module that contains the logic common for both Android and iOS applications, the code you share between platforms.
+
+This `shared` module is also where you write your Compose Multiplatform code.
+In `shared/src/commonMain/kotlin/App.kt`, you can find the shared root `@Composable` function for your app.
 
-**This is also where you will write your Compose Multiplatform code**.
+It uses Gradle as the build system. You can add dependencies and change settings in `shared/build.gradle.kts`.
+The shared module builds into an Android library and an iOS framework.
 
-The shared root `@Composable` function for your app lives in `shared/src/commonMain/kotlin/App.kt`.
+### androidApp
 
-`shared` uses Gradle as the build system. You can add dependencies and change settings in `shared/build.gradle.kts`. The shared module builds into an Android library and an iOS framework.
+This is a Kotlin module that builds into an Android application. It uses Gradle as the build system.
+The `androidApp` module depends on and uses the shared module as a regular Android library.
 
-### `androidApp`
+### iosApp
 
-This Kotlin module that builds into an Android application. It uses Gradle as the build system. The androidApp module depends on and uses the shared module as a regular Android library.
+This is an Xcode project that builds into an iOS application.
+It depends on and uses the shared module as a CocoaPods dependency.
 
-### `iosApp`
-This is the Xcode project that builds into an iOS application. It depends on and uses the shared module as a CocoaPods dependency.
+## Run your application
 
-## Running your application
+### On Android
 
-## Android
 To run your application on an Android emulator:
 
-- Create an [Android virtual device](https://developer.android.com/studio/run/managing-avds#createavd).
-- Select the `androidApp` run configuration.
-- Select your target device and press **Run**.
+1. Create an [Android virtual device](https://developer.android.com/studio/run/managing-avds#createavd).
+2. In the list of run configurations, select `androidApp`.
+3. Choose your virtual device and click **Run**:
 
-<img src="readme_images/run_on_android.png" height="60px">
-<img src="readme_images/android_app_running.png" height="200px">
+   <img src="readme_images/run_on_android.png" height="60px">
+   
+   <img src="readme_images/android_app_running.png" height="200px">
 
 <details>
-  <summary>Using Gradle</summary>
+  <summary>Alternatively, use Gradle</summary>
```

Comment:
This is a collapsed spoiler. Not sure if this grammatical construction works there?
Anyway, no strong opinions

Reply:
I think it works with the overall flow of the instructions